tests/l2/test_l2_config.py

```diff
@@ -0,0 +1,74 @@
+"""
+Tests related to L2 configuration
+"""
+import logging
+import pytest
+import time
+from tests.common.helpers.assertions import pytest_expect
+from tests.common.checkpoint import (
+    create_checkpoint,
+    delete_checkpoint,
+    rollback,
+)
+
+logger = logging.getLogger(__name__)
+
+DUT_SONIC_IMAGE = "/tmp/sonic_image.bin"
+DUT_SONIC_CFG = "/tmp/sonic-cfg.json"
+DUT_ORIGIN_CFG = "/tmp/sonic-cfg-orig.json"
+
+DUMP_DB_CMD = "sonic-db-dump -y -n CONFIG_DB -k 'TELEMETRY|*'"
+DB_DUMP_IGNORE = ["expireat"]
+
+pytestmark = [
+    pytest.mark.topology('any')
+]
+
+
+def compare_db_dump(db_before, db_after):
+    """
+    Compare two db dumps and after the L2 configuration steps.
+    Assert there are no additional configuration coming from minigraph.
```

Comment:
I don't understand this test, how do we introduce additional configuration coming from minigraph?